util.py

```diff
@@ -498,7 +498,7 @@ def guess_rmantree_initial():
     rmantree = rmantree_from_env()
     if rmantree != '':
         vstr = rmantree.split('-')[-1]
-        vf = float(vstr.strip('/'))
```

Comment:
Could be.  Problem is that first \ would cause Python to ignore the quotation after it so shouldn't it be / followed by two forward slashes?  I can't even write it down right here cause the comment box erases one of the forward slashes.